src/Synapse/Synapse.Autorest/test/New-AzSynapseKustoPool.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+Describe 'New-AzSynapseKustoPool' {
```

Comment:
Session record for New-*KustoPool?

Reply:
As I offline discuss with Xiaogang, I will mark New/Remove-*KustoPool test case as "LiveOnly". Because customization of New cmd use already exist cmd "Get-*Workspace" which can't be mocked.

Reply:
Thanks for your explanation